ui/src/views/access-point/cname-step/cname-step.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Button,
+  Card,
+  DisplayText,
+  Grid,
+  Icon,
+  IconButton,
+  SpinnerDot,
+  Stepper,
+  Text,
+} from '@/components';
+import { useState } from 'react';
+
+export const CNAMEStep: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const { prevStep, nextStep } = Stepper.useContext();
+
+  const handleContinueClick = (): void => {
+    //TODO add CNAME creation handler
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      nextStep();
+    }, 3000);
+  };
+
+  return (
+    <Card.Container css={{ width: '$107h' }}>

```

Comment:
I would split this into `Header` and `Body` fragments inside this same file to keep the render smaller.